### ISSUE TITLE
Buggy flow interfaces

### ISSUE THREAD

**Issue by nifgraup** [alangpierce/sucrase#407]
in/bug.js:
```
class foo implements Bar {
  baz = () => {

  }
}
```
`sucrase in -d out -t flow`

out/bug.js:
```
class foo  {constructor() { null.prototype.__init.call(this); }
  __init() {this.baz = () => {

  }}
}
```

should be:
```
class foo {constructor() { foo.prototype.__init.call(this); }
  __init() {this.baz = () => {

  }}
}
```

**Comment by alangpierce** [comment]
Thanks for reporting! Looks like a straightforward bug in the Flow parser (which is why it doesn't happen in TypeScript). Fix should be released in the next day or two.

**Comment by alangpierce** [comment]
@nifgraup sorry for the delay, I just published as 3.9.6.
